Plugin~/WebRTCPlugin/GraphicsDevice/IGraphicsDevice.h

```diff
@@ -0,0 +1,20 @@
+﻿#pragma once
+
+namespace WebRTC {
+
+struct ITexture2D;
+
+class IGraphicsDevice {
+public:
+
+    IGraphicsDevice();
+    virtual ~IGraphicsDevice() = 0;
+
+    virtual void ShutdownV() = 0;
+    virtual ITexture2D* CreateEncoderInputTextureV(uint32_t width, uint32_t height) = 0;
+    virtual ITexture2D* CreateEncoderInputTextureV(uint32_t width, uint32_t height, void* nativeTexturePtr) = 0;
+    virtual void* GetNativeDevicePtrV() = 0;
```

Comment:
It seems better this method is declared as `const`

Reply:
Yeah, I thought the same way, but `NV_ENC_OPEN_ENCODE_SESSION_EX_PARAMS.device` doesn't accept const, so it won't work

Reply:
Got it